libs/vertexai/langchain_google_vertexai/embeddings.py

```diff
@@ -215,7 +222,10 @@ def _get_multimodal_embeddings_with_retry(
         return embeddings
 
     def _get_text_embeddings_with_retry(
-        self, texts: List[str], embeddings_type: Optional[str] = None
+        self,
+        texts: List[str],
+        embeddings_type: Optional[str] = None,
+        dimensions: Optional[int] = None,
```

Comment:
can we stick to the argument's naming the same as Vertex uses, please? i.e., `output_dimensionality` instead of `dimensions`?